src/displayapp/screens/Dice.cpp

```diff
@@ -0,0 +1,219 @@
+#include "Dice.h"
+
+#include "Screen.h"
+#include "Symbols.h"
+#include "lvgl/lvgl.h"
+
+using namespace Pinetime::Applications::Screens;
+constexpr uint8_t Dice::diceSizes[];
+
+static void num_dice_inc_event_handler(lv_obj_t* obj, lv_event_t event) {
+  auto dice = static_cast<Dice*>(obj->user_data);
+  dice->numDiceIncBtnEventHandler(event);
+}
+
+static void num_dice_dec_event_handler(lv_obj_t* obj, lv_event_t event) {
+  auto dice = static_cast<Dice*>(obj->user_data);
+  dice->numDiceDecBtnEventHandler(event);
+}
+
+static void dice_size_inc_event_handler(lv_obj_t* obj, lv_event_t event) {
+  auto dice = static_cast<Dice*>(obj->user_data);
+  dice->diceSizeIncBtnEventHandler(event);
+}
+
+static void dice_size_dec_event_handler(lv_obj_t* obj, lv_event_t event) {
+  auto dice = static_cast<Dice*>(obj->user_data);
+  dice->diceSizeDecBtnEventHandler(event);
+}
+
+static void roll_event_handler(lv_obj_t* obj, lv_event_t event) {
+  auto dice = static_cast<Dice*>(obj->user_data);
+  dice->rollBtnEventHandler(event);
+}
+
+void Dice::createScreen() {
+  resultCont = lv_cont_create(lv_scr_act(), nullptr);
+  lv_obj_set_size(resultCont, 240, 80);
+  lv_obj_set_pos(resultCont, 0, 0);
+  lv_cont_set_layout(resultCont, LV_LAYOUT_COLUMN_LEFT);
+  lv_obj_set_style_local_pad_all(resultCont, LV_CONT_PART_MAIN, LV_STATE_DEFAULT, 10);
+
+  resultTotalLabel = lv_label_create(resultCont, nullptr);
+  lv_obj_set_size(resultTotalLabel, 220, 30);
+  lv_obj_set_style_local_text_color(resultTotalLabel, LV_LABEL_PART_MAIN, LV_STATE_DEFAULT, LV_COLOR_BLACK);
+  lv_label_set_text_static(resultTotalLabel, "");
+
+  resultIndRollsLabel = lv_label_create(resultCont, nullptr);
+  lv_label_set_long_mode(resultIndRollsLabel, LV_LABEL_LONG_SROLL_CIRC);
+  lv_obj_set_size(resultIndRollsLabel, 220, 30);
+  lv_obj_set_style_local_text_color(resultIndRollsLabel, LV_LABEL_PART_MAIN, LV_STATE_DEFAULT, LV_COLOR_BLACK);
+  lv_label_set_text_static(resultIndRollsLabel, "");
+
+  configCont = lv_cont_create(lv_scr_act(), nullptr);
+  lv_obj_set_size(configCont, 240, 154);
+  lv_obj_set_pos(configCont, 0, 86);
+  lv_obj_set_style_local_bg_color(configCont, LV_CONT_PART_MAIN, LV_STATE_DEFAULT, LV_COLOR_BLACK);
+
+  numDiceIncBtn = lv_btn_create(configCont, nullptr);
+  numDiceIncBtn->user_data = this;
+  lv_obj_set_event_cb(numDiceIncBtn, num_dice_inc_event_handler);
+  lv_obj_set_size(numDiceIncBtn, 76, 52);
+  lv_obj_align(numDiceIncBtn, NULL, LV_ALIGN_IN_TOP_LEFT, 0, 0);
+  lv_obj_set_style_local_value_str(numDiceIncBtn, LV_BTN_PART_MAIN, LV_STATE_DEFAULT, "+");
+
+  numDiceLabelCont = lv_cont_create(configCont, nullptr);
+  lv_obj_set_size(numDiceLabelCont, 76, 50);
+  lv_obj_align(numDiceLabelCont, NULL, LV_ALIGN_IN_LEFT_MID, 0, 0);
```

Comment:
multiple `Use nullptr` warnings

Reply:
@Riksu9000 Fixed this in https://github.com/JF002/InfiniTime/pull/565/commits/75c7e280842f08b91be91de121d600ec5f87defd